packages/page/src/Infrastructure/Doctrine/Repository/PageRepository.php

```diff
@@ -222,10 +221,18 @@ public function reorderOneBy(array $filters, int $position): void
         } elseif ($movementSteps < 0) {
             $this->entityRepository->moveDown($page, \abs($movementSteps));
         }
+    }
+
+    public function moveOneBy(array $sourceFilters, array $destinationFilters): void
+    {
+        $sourcePage = $this->getOneBy($sourceFilters);
+        $destinationPage = $this->getOneBy($destinationFilters);
 
-        if (true !== $this->entityRepository->verify()) {
```

Comment:
I removed this from the `reorderOneBy` method, because the tree is always invalid before flushing, so we always did call recover in a false positive way